.github/workflows/main.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
       - name: Install Python packages
         run: |
-           pip install pytest==3.10.1; pip install pytest-xdist==1.26.1; pip install scipy==1.2.1; pip install nose==1.3.7
+           pip install pytest==3.10.1; pip install pytest-xdist==1.26.1; pip install scipy==1.2.1; pip install nose==1.3.7; pip install mock
```

Comment:
Could specify the exact version here as done for the other packages but not a major issue (I doubt they are going to release a python 2 package version which breaks behaviour).

Reply:
Pinned to 3.0.5